### ISSUE TITLE
Xperia 10 Not supported, trying to flash custom firmware results in corruption, reflashing it is impossible since it is not added to the Github list.

### ISSUE THREAD

**Issue by ToastedToast128** [Androxyde/Flashtool#112]
I installed the firmware from here: (https://forum.xda-developers.com/xperia-10/development/xperia-10-kernel-t3920073), and I corrupted my phone. Luckily I had a backup of my firmware, but it is impossible to re-install.

**Comment by jetharius** [comment]
Similar problem here with the Xperia 10.  Rolling back to older version of flashtool to bypass the device list results in a failure to detect the device in flash mode (instead detects as connected device with debugging disabled. v0.9.18).  Latest v0.9.25 user cannot select ftf images until device filter is populated (i assume), and no I31XX or I41XX xperia 10 options exist in the device list.

**Comment by ToastedToast128** [comment]
Maybe an even older version of flashtool will work. otherwise I think I just lost $300

**Comment by jetharius** [comment]
If you find one let me know, Im guessing it will just be a waiting game until the flashtool team or EMMA start support for the Xperia 10.  Phone has only been out for few months now.  As long as you can get your device into fastboot mode or flash mode, it will just be bricked until then so I wouldnt consider it a total loss.  Alternatively you can try fastboot flashing each file of the ftf image one by one...  I personally am not confident on how to do that, so I'm stuck waiting.

**Comment by jetharius** [comment]
Alternatively a workaround from the flashtool team would be grand.

**Comment by ToastedToast128** [comment]
I'll try flashing the individual files tonight, otherwise I'm will have to hope that best buy takes the phone back (unlikely, but I just bought the phone yesterday)

**Comment by Bin4ry** [comment]
Hey,

i don't have the phone so i cannot create device support and there is no
pull request to integrate it.
Phone is not lost, maybe someone can create the needed device files for you
or you can try to flash the phone with someone who has access to EMMA
(Sonys internal flashing software).

Sorry that i currently cannot help further.

Best Regards
Andreas

Am Di., 7. Mai 2019 um 20:50 Uhr schrieb ToastedToast128 <
notifications@github.com>:

**Comment by ToastedToast128** [comment]
EMMA cannot recognize the device.

**Comment by ToastedToast128** [comment]
YES, I FOUND A WORKAROUND!!! Using NewFlasher (https://forum.xda-developers.com/crossdevice-dev/sony/progress-newflasher-xperia-command-line-t3619426) and XperiFirm (https://forum.xda-developers.com/crossdevice-dev/sony/pc-xperifirm-xperia-firmware-downloader-t2834142), i was able to recover the device! This is good news for anyone trying to mess with their device like I did

**Comment by jetharius** [comment]
Aye, worked for me.  Good find.  Should sticky this on the XDA forums for
this device until flashtool and EMMA gain support.

**Comment by Androxyde** [comment]
At the moment, flashtool does not support Xperia 10. It will as soon as I get the device. I have no ETA

**Comment by psycholyzern** [comment]
I was trying to fix my Xperia C and stumbled upon this issue. I checked XperiFirm and there is no firmware for this phone. After some Googling, I found [Flashtool by Androxyde](https://github.com/Androxyde/Flashtool/). I successfully flashing ftf file with it.

I'm posting this just because someone might ended up here just like I was wiht the same problem.
